### PR TITLE
Issue #681 Replace references to virtualbox in tests by hyperkit

### PR DESCRIPTION
**Pull request by jsliacan**

### CODE REVIEW

**Comment by praveenkumar** [comment]
retest this please.

File: test/integration/features/config.feature [hunk 1]
```diff
@@ -74,7 +74,7 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
         @darwin
         Examples:
             | property                             | value1 | value2 |
-            | skip-check-virtualbox-installed      | true   | false  |
+            | skip-check-hyperkit-installed        | true   | false  |
```
Comment (praveenkumar) [inline test/integration/features/config.feature]: @jsliacan this looks like some extra whitespace at the end of `skip-check-hyperkit-installed`, `warn-check-hyperkit-installed` and `hyperkit`

Comment (jsliacan) [reply test/integration/features/config.feature]: I reformatted all tables in `config.feature`, but some of those you pointed out didn't change. It doesn't look like there are extra white spaces though if you look here (I use an auto-fmt for feature files in emacs): https://github.com/jsliacan/crc/blob/killvboxrefs/test/integration/features/config.feature

Comment (cfergeau) [reply test/integration/features/config.feature]: The formatting is done with whitespaces, the ones which are highlighted in this diff is the 'ed' before which is changed to '  '. Change looks good to me, especially as CI passes ;)

**Pull request merged**